tsconfig.json

```diff
@@ -5,7 +5,8 @@
         "outDir": "out",
         "lib": [
             "es6",
-            "dom"
+            "dom",
+            "ES2020.Promise"
```

Comment:
I had to add this to be able to use the `Promise.allSettled` method. Do we have to add polyfills for this? How do we know what we can or can't use here?